tox.ini

```diff
@@ -114,7 +114,7 @@ commands =
 [testenv:integration]
 description = Run integration tests
 deps =
-    juju==2.9.42.4
+    juju
```

Comment:
I think the latest version for juju 2.9/stable used in integration tests is `juju==2.9.46.1`